trifle/program/src/state/transfer_effects.rs

```diff
@@ -22,6 +22,7 @@ impl From<u16> for TransferEffects {
     }
 }
 
+#[allow(clippy::from_over_into)]
```

Comment:
Is this a deliberate choice to implement `Into`, instead of `From`, which gives you the `Into` implementation automatically?